2023/sg-election-votes/index.html

```diff
@@ -0,0 +1,40 @@
+<html lang="en">
+    <head>
+        <title>2023 Student Government Election Results</title>
```

Comment:
I think this title may work better: `Student Government Election Votes: 2023`. Could you apply it to the HTML page and chart?

Reply:
Went with Student Government Election Results: 2023